geoipupdate/geoip.py

```diff
@@ -1,37 +1,41 @@
 import hashlib
 import os
 import urllib
-import gzip
+import tarfile
 import shutil
 import logging
 
 
-MAXMIND_URL = 'http://geolite.maxmind.com/download/geoip/database/'
-MAXMIND_FILENAME = 'GeoLite2-Country.mmdb.gz'
+license_key = os.environ['LICENSE-KEY']
+MAXMIND_URL = 'https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country' \
+              '&license_key={key}&suffix=tar.gz'.format(key=license_key)
+MAXMIND_SHA256_URL = 'https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country' \
+                     '&license_key={key}&suffix=tar.gz.sha256'.format(key=license_key)
+MAXMIND_FILENAME = 'GeoLite2-Country_20200324.tar.gz'
```

Comment:
Can we remove the date stamp from the file name? Because the date will not be the same for each run.